flatpak/org.jabref.jabref.json

```
@@ -0,0 +1,33 @@
+{
+    "app-id" : "org.jabref.jabref",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "JabRef",
+    "modules" : [
+        {
+            "name" : "JabRef",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "tar -xvzf flatpak/JabRef-portable_linux.tar.gz -C /app/"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "path": "JabRef-portable_linux.tar.gz"
```

```suggestion
                    "url": "http://builds.jabref.org/master/JabRef-portable_linux.tar.gz",
                    "sha256": "be69feee0536fa8500dda867290dbeeeda6191c60e636a0b6e254fbaac02d471"
```

The checksum might not be updated to the latest right now...